tests/integration/api-breaks/IndexLayerClient.test.ts

```diff
@@ -0,0 +1,214 @@
+  });
+
+  it("Shoud be initialized with IndexLayerClientParams", async () => {
+    const layerClient = new IndexLayerClient({
+      catalogHrn: HRN.fromString("hrn:here:data:::test-hrn"),
+      layerId: "test-layed-id",
+      settings: new OlpClientSettings({
+        environment: "here",
+        getToken: () => Promise.resolve("test-token-string")
+      })
+    });
+    assert.isDefined(layerClient);
+    expect(layerClient).to.be.instanceOf(IndexLayerClient);
+
+    assert.isFunction(layerClient.getData);
+    assert.isFunction(layerClient.getPartitions);
+    assert.isDefined(layerClient.hrn);
+    assert.isDefined(layerClient.layerId);
+    assert.isDefined(layerClient.settings);
+  });
+
+  it("getPartitions method with IndexQueryRequest", async () => {
+    const layerClient = new IndexLayerClientTest({
+      catalogHrn: HRN.fromString("hrn:here:data:::test-hrn"),
+      layerId: "test-layed-id",
+      settings: new OlpClientSettings({
+        environment: "here",
+        getToken: () => Promise.resolve("test-token-string")
+      })
+    });
+
+    const response = layerClient.getPartitions(new IndexQueryRequest());
+    assert.isDefined(response);
+  });
+
+  it("getPartitions method with IndexQueryRequest and abort signal", async () => {
+    const layerClient = new IndexLayerClientTest({
+      catalogHrn: HRN.fromString("hrn:here:data:::test-hrn"),
+      layerId: "test-layed-id",
+      settings: new OlpClientSettings({
+        environment: "here",
+        getToken: () => Promise.resolve("test-token-string")
+      })
+    });
+
+    const abortController = new AbortController();
+
+    const response = layerClient.getPartitions(
+      new IndexQueryRequest(),
+      abortController.signal
+    );
+    assert.isDefined(response);
+  });
+
+  it("getData method method with model", async () => {
+    const layerClient = new IndexLayerClientTest({
+      catalogHrn: HRN.fromString("hrn:here:data:::test-hrn"),
+      layerId: "test-layed-id",
+      settings: new OlpClientSettings({
+        environment: "here",
+        getToken: () => Promise.resolve("test-token-string")
+      })
+    });
+
+    const mockedModel = {
+      id: "8c0e5ac9-b036-4365-8820-dfcba64588fc",
+      size: 111928,
+      checksum: "448a33cd65c47bed1eeb4d72e7fa022c95a41158",
+      timestamp: 1551981674191,
+      hour_from: 1506402000000,
+      tile_id: 377894442,
+      crc: null
+    };
+    const response = await layerClient.getData(mockedModel);
+
+    assert.isDefined(response);
+  });
+
+  it("getData method method with model and abort signal", async () => {
+    const layerClient = new IndexLayerClientTest({
+      catalogHrn: HRN.fromString("hrn:here:data:::test-hrn"),
+      layerId: "test-layed-id",
+      settings: new OlpClientSettings({
+        environment: "here",
+        getToken: () => Promise.resolve("test-token-string")
+      })
+    });
+
+    const abortController = new AbortController();
+
+    const mockedModel = {
```

Comment:
Please, add case when model is empty. As we see, IndexApi.Index do not have any required props

Reply:
Added Test for getData method with empty model